### DISCUSSION TITLE
Default Avatar Option

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Authors#1300] [Feature Requests]
By default, PublishPress Author will pull from Gravatar. 

I dont want my users in my website use image from Gravatar. But I have many users, it will be wasting time to change one by one.

It will be good if we have default fallback avatar option that maybe can be **set generally** in setting not setup one by one.
![image](https://user-images.githubusercontent.com/83811723/228776898-c56f2e95-3e56-4496-ba8b-b93c2d0454a0.png)

**Comment by stevejburge** [comment]
More:

> The plugin is not pulling in the Gravitar default image https://secure.gravatar.com/avatar/?s=64&d=mm&r=g for authors where Gravitar is selected but there is no Gravitar for that author.
